Jamulus.pro

```diff
@@ -1,11 +1,26 @@
-VERSION = 3.6.2git
+VERSION = 3.6.2dev
 
 # use target name which does not use a captital letter at the beginning
 contains(CONFIG, "noupcasename") {
     message(The target name is jamulus instead of Jamulus.)
     TARGET = jamulus
 }
 
+#allow detailed version info for intermediate builds
+contains(CONFIG, "official_release_version") {
```

Comment:
Why to use a config flag for it? Couldn't you parse the VERSION and if you find a "dev" then you know it is a developer version and if no "dev" is present, you know it is an official release version.

Reply:
What I do when I create a new release is that I modify the ChangeLog and then I update the VERSION in the .pro file and create a Git commit from it. Then I apply the release tag. Then ann0see's automation detects that it is an official release version and starts his scripts. In that Git commit, the VERSION is set correctly to a release version number (which matches the entry in the ChangeLog).

Reply:
> Then ann0see's automation detects that it is an official release version and starts his scripts [...]

Exactly. If you merge @nefarius2001 changes, the latest tag will not be applied to this commit and you'll still need to add it automatically. (Just wanted to add this).

Reply:
@corrados I had concerns, that somehow it might be forgotten/screwed up to add the "dev" appendix after removing it for the release. But it makes the job much easier, so lets go that road

Reply:
@ann0see "the latest tag will not be applied to this commit" -> What do you mean by this? Do you refer to your Git automation scripts which do a release if a release tag is applied to the repo? But here we are talking about the version number shown in the Jamulus software. If the "dev" is not there, nothing happens (at least when looking at the code after the latest change).

Reply:
> I had concerns, that somehow it might be forgotten/screwed up to add the "dev" appendix after removing it for the release.

The is the very first thing I am doing after finishing a release. You can take a look at the Git history if you like.

Reply:
>  Do you refer to your Git automation scripts which do a release if a release tag is applied to the repo?

Yes. That was just an addition to your answer. I wasn't sure you were aware of this.